### PR TITLE
enh(getting started) - fr monitor first linux host with snmp

### PR DESCRIPTION
**Pull request by cedricmeschin**
## Description

Like new monitoring linux page in english, the same in french

## Target serie

- [ ] 20.04.x
- [ ] 20.10.x
- [ ] 21.04.x
- [X] 21.10.x
- [X] 22.04.x (master)

### CODE REVIEW

File: fr/getting-started/monitor-linux-server-with-snmp.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
+---
+id: monitor-linux-server-with-snmp
+title: Supervise ton premier serveur Linux
+---
+
+## Superviser un serveur Linux avec SNMP
+
+Dans ce tutoriel, nous partons du principe que votre plate-forme Centreon est installée et fonctionne correctement, et que vous disposez au moins d'une [Centreon IT 100 Edition](IT100.html) qui fournit les Plugin Packs Centreon (votre [license](../administration/licenses.html) est déjà en place).
```
Comment (cg-tw) [inline fr/getting-started/monitor-linux-server-with-snmp.md:8]:
```suggestion
Dans ce tutoriel, nous partons du principe que votre plate-forme Centreon est installée et fonctionne correctement, et que vous disposez au moins d'une édition [Centreon IT 100](IT100.html) qui fournit les Plugin Packs Centreon (votre [licence](../administration/licenses.html) est déjà en place).

```

File: fr/getting-started/monitor-linux-server-with-snmp.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
+---
+id: monitor-linux-server-with-snmp
+title: Supervise ton premier serveur Linux
+---
+
+## Superviser un serveur Linux avec SNMP
+
+Dans ce tutoriel, nous partons du principe que votre plate-forme Centreon est installée et fonctionne correctement, et que vous disposez au moins d'une [Centreon IT 100 Edition](IT100.html) qui fournit les Plugin Packs Centreon (votre [license](../administration/licenses.html) est déjà en place).
+
+Votre serveur sera supervisé à l'aide de [Linux SNMP Plugin Pack](../integrations/plugin-packs/procedures/operatingsystems-linux-snmp.html). (Plus d'informations sur les Plugin Packs [here](../monitoring/pluginpacks.html)).
```
Comment (cg-tw) [inline fr/getting-started/monitor-linux-server-with-snmp.md:10]:
```suggestion
Votre serveur sera supervisé à l'aide du Plugin Pack [Linux SNMP](../integrations/plugin-packs/procedures/operatingsystems-linux-snmp.html). (Plus d'informations sur les Plugin Packs [ici](../monitoring/pluginpacks.html)). 

```

File: fr/getting-started/monitor-linux-server-with-snmp.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+title: Supervise ton premier serveur Linux
+---
+
+## Superviser un serveur Linux avec SNMP
+
+Dans ce tutoriel, nous partons du principe que votre plate-forme Centreon est installée et fonctionne correctement, et que vous disposez au moins d'une [Centreon IT 100 Edition](IT100.html) qui fournit les Plugin Packs Centreon (votre [license](../administration/licenses.html) est déjà en place).
+
+Votre serveur sera supervisé à l'aide de [Linux SNMP Plugin Pack](../integrations/plugin-packs/procedures/operatingsystems-linux-snmp.html). (Plus d'informations sur les Plugin Packs [here](../monitoring/pluginpacks.html)). 
+
+## Pré-requis
```
Comment (cg-tw) [inline fr/getting-started/monitor-linux-server-with-snmp.md:12]:
```suggestion
## Prérequis

```

File: fr/getting-started/monitor-linux-server-with-snmp.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+
+### sur le serveur Linux que vous souhaiter superviser
```
Comment (cg-tw) [inline fr/getting-started/monitor-linux-server-with-snmp.md:14]:
```suggestion
### Sur le serveur Linux que vous souhaiter superviser

```

File: fr/getting-started/monitor-linux-server-with-snmp.md [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+
+La première étape consiste à activer et à configurer l'agent SNMP sur votre hôte à superviser.
```
Comment (cg-tw) [inline fr/getting-started/monitor-linux-server-with-snmp.md:16]:
```suggestion
La première étape consiste à activer et à configurer l'agent SNMP sur l'hôte à superviser.

```

File: fr/getting-started/monitor-linux-server-with-snmp.md [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+Veuillez vous référer à la documentation de votre distribution Linux pour savoir comment configurer l'agent SNMP.
+
+Trouvez ci-dessous un fichier de configuration minimaliste de snmpd.conf/net-snmp :
```
Comment (cg-tw) [inline fr/getting-started/monitor-linux-server-with-snmp.md:19]:
```suggestion
Trouvez ci-dessous un fichier de configuration **snmpd.conf/net-snmp** minimaliste :

```

File: fr/getting-started/monitor-linux-server-with-snmp.md [hunk 7]
```diff
@@ -0,0 +1,108 @@
...
+  - remplacez **my-snmp-community** par la valeur correcte pour votre environnement.
```
Comment (cg-tw) [inline fr/getting-started/monitor-linux-server-with-snmp.md:20]:
```suggestion
  - Remplacez **my-snmp-community** par la valeur correspondant à votre environnement.

```

File: fr/getting-started/monitor-linux-server-with-snmp.md [hunk 8]
```diff
@@ -0,0 +1,108 @@
...
+  - Ajouter la ligne **view centreon included .1.3.6.1** pour avoir accès à toutes les informations de la MIB requises par le plugin
```
Comment (cg-tw) [inline fr/getting-started/monitor-linux-server-with-snmp.md:21]:
```suggestion
  - Ajoutez la ligne **view centreon included .1.3.6.1** pour avoir accès à toutes les informations de la MIB requises par le plugin

```

File: fr/getting-started/monitor-linux-server-with-snmp.md [hunk 9]
```diff
@@ -0,0 +1,108 @@
...
+```shell
+systemctl restart snmpd
+systemctl enable snmpd
+```
+
+> Le serveur cible doit être accessible depuis le collecteur Centreon sur le port SNMP UDP/161.
+
+### Sur le collecteur
+
+1. Connectez-vous à votre collecteur en SSH et installez le plugin SNMP Linux (voir la [procédure de surveillance pour le **Linux SNMP** Plugin Pack](../integrations/plugin-packs/procédures/operatingsystems-linux-snmp.html) pour plus d'informations) :
```
Comment (cg-tw) [inline fr/getting-started/monitor-linux-server-with-snmp.md:55]:
```suggestion
1. Connectez-vous à votre collecteur en SSH et installez le plugin SNMP Linux (voir la [procédure de surveillance pour le Plugin Pack **Linux SNMP**](../integrations/plugin-packs/procédures/operatingsystems-linux-snmp.html) pour plus d'informations) :

```

File: fr/getting-started/monitor-linux-server-with-snmp.md [hunk 10]
```diff
@@ -0,0 +1,108 @@
...
+
+   ```shell
+   yum install centreon-plugin-Operatingsystems-Linux-Snmp
+   ```
+
+2. Dans l'interface web, allez dans **Configuration > Plugin Packs** et installez le Plugin Pack **Linux SNMP** :
```
Comment (cg-tw) [inline fr/getting-started/monitor-linux-server-with-snmp.md:61]:
```suggestion
2. Dans l'interface web, allez à la page **Configuration > Plugin Packs** et installez le Plugin Pack **Linux SNMP** :

```

File: fr/getting-started/monitor-linux-server-with-snmp.md [hunk 11]
```diff
@@ -0,0 +1,108 @@
...
+
+   ![image](../assets/getting-started/quick_start_linux_0.gif)
+
+
+## Configurer l'hôte et déployer la configuration
+
+1. Allez dans **Configuration > Hôtes > Hôtes** et cliquez sur **Ajouter** :
```
Comment (cg-tw) [inline fr/getting-started/monitor-linux-server-with-snmp.md:68]:
```suggestion
1. Allez à la page **Configuration > Hôtes > Hôtes** et cliquez sur **Ajouter** :

```

File: fr/getting-started/monitor-linux-server-with-snmp.md [hunk 12]
```diff
@@ -0,0 +1,108 @@
...
+
+   ![image](../assets/getting-started/quick_start_linux_1.gif)
+
+2. Remplissez les informations suivantes :
+
+   * Le nom du serveur (1)
+   * Une description de celui-ci (2)
+   * L'adresse IP (3)
+   * La communauté SNMP et sa version (4)
+   * Sélectionner le collecteur approprié (laissez "Central" si vous n'avez pas d'autre collecteur) (5)
```
Comment (cg-tw) [inline fr/getting-started/monitor-linux-server-with-snmp.md:78]:
```suggestion
   * Sélectionnez le collecteur désiré (laissez **Central** si vous n'avez pas d'autre collecteur) (5)

```

File: fr/getting-started/monitor-linux-server-with-snmp.md [hunk 13]
```diff
@@ -0,0 +1,108 @@
...
+
+1. Allez dans **Configuration > Hôtes > Hôtes** et cliquez sur **Ajouter** :
+
+   ![image](../assets/getting-started/quick_start_linux_1.gif)
+
+2. Remplissez les informations suivantes :
+
+   * Le nom du serveur (1)
+   * Une description de celui-ci (2)
+   * L'adresse IP (3)
```
Comment (cg-tw) [inline fr/getting-started/monitor-linux-server-with-snmp.md:76]:
```suggestion
   * L'adresse IP du serveur (3)

```

File: fr/getting-started/monitor-linux-server-with-snmp.md [hunk 14]
```diff
@@ -0,0 +1,108 @@
...
+   * La communauté SNMP et sa version (4)
+   * Sélectionner le collecteur approprié (laissez "Central" si vous n'avez pas d'autre collecteur) (5)
+
+3. Cliquez sur **+ Ajouter une novelle entrée** dans le champ **Modèles** (6), puis sélectionnez le modèle **OS-Linux-SNMP-custom** (7) dans la liste :
+
+   ![image](../assets/getting-started/quick_start_linux_2.png)
+
+4. Cliquez sur **Sauvegarderve** (8). Votre équipement a été ajouté à la liste des hôtes :
```
Comment (cg-tw) [inline fr/getting-started/monitor-linux-server-with-snmp.md:84]:
```suggestion
4. Cliquez sur **Sauvegarder** (8). Votre équipement a été ajouté à la liste des hôtes :

```

File: fr/getting-started/monitor-linux-server-with-snmp.md [hunk 15]
```diff
@@ -0,0 +1,108 @@
...
+
+   ![image](../assets/getting-started/quick_start_linux_3.png)
+
+5. Allez dans **Configuration > Services > Services par hôte**. Un ensemble d'indicateurs a été créé automatiquement.
```
Comment (cg-tw) [inline fr/getting-started/monitor-linux-server-with-snmp.md:88]:
```suggestion
5. Allez à la page **Configuration > Services > Services par hôte**. Un ensemble d'indicateurs a été créé automatiquement.

```

File: fr/getting-started/monitor-linux-server-with-snmp.md [hunk 16]
```diff
@@ -0,0 +1,108 @@
...
+
+   ![image](../assets/getting-started/quick_start_linux_4.png)
+
+   Vous pouvez également utiliser le raccourci situé à côté du nom de l'hôte pour accéder directement à **Configuration > Services > Services par hôte**. La liste sera filtrée par le nom de l'hôte :
```
Comment (cg-tw) [inline fr/getting-started/monitor-linux-server-with-snmp.md:92]:
```suggestion
   Vous pouvez également utiliser le raccourci situé à côté du nom de l'hôte pour accéder directement à la page **Configuration > Services > Services par hôte**. La liste sera filtrée par le nom de l'hôte :

```

File: fr/getting-started/monitor-linux-server-with-snmp.md [hunk 17]
```diff
@@ -0,0 +1,108 @@
...
+
+   ![image](../assets/getting-started/quick_start_linux_4b.png)
+
+   ![image](../assets/getting-started/quick_start_linux_5.png)
+
+6. [Déployer la configuration](first-supervision#deploying-a-configuration).
```
Comment (cg-tw) [inline fr/getting-started/monitor-linux-server-with-snmp.md:98]:
```suggestion
6. [Déployez la configuration](first-supervision#deploying-a-configuration).

```

File: fr/getting-started/monitor-linux-server-with-snmp.md [hunk 18]
```diff
@@ -0,0 +1,108 @@
...
+
+7. Allez dans **Surveillance > Statut des ressources** et sélectionnez **Toutes** dans le filtre **Statut des ressources**. Dans un premier temps, les ressources apparaissent avec le statut **En attente**, ce qui signifie qu'aucun contrôle n'a encore été exécuté :
```
Comment (cg-tw) [inline fr/getting-started/monitor-linux-server-with-snmp.md:100]:
```suggestion
7. Allez à la page **Surveillance > Statut des ressources** et sélectionnez **Toutes** dans le filtre **Statut des ressources**. Dans un premier temps, les ressources apparaissent avec le statut **En attente**, ce qui signifie qu'aucun contrôle n'a encore été exécuté :

```

**Review by cg-tw — CHANGES_REQUESTED** [review]

**Review by pkriko — DISMISSED** [review]

File: fr/getting-started/monitor-linux-server-with-snmp.md [hunk 19]
```diff
@@ -0,0 +1,108 @@
+---
+id: monitor-linux-server-with-snmp
+title: Supervise ton premier serveur Linux
```
Comment (cg-tw) [inline fr/getting-started/monitor-linux-server-with-snmp.md:3]:
```suggestion
title: Superviser un serveur Linux

```

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: fr/getting-started/monitor-linux-server-with-snmp.md [hunk 20]
```diff
@@ -0,0 +1,108 @@
...
+Votre serveur sera supervisé à l'aide du Plugin Pack [Linux SNMP](../integrations/plugin-packs/procedures/operatingsystems-linux-snmp.html). (Plus d'informations sur les Plugin Packs [ici](../monitoring/pluginpacks.html)). 
+
+## Prérequis
+
+### Sur le serveur Linux que vous souhaitez superviser
+
+La première étape consiste à activer et à configurer l'agent SNMP sur l'hôte à superviser.
+Veuillez vous référer à la documentation de votre distribution Linux pour savoir comment configurer l'agent SNMP.
+
+Trouvez ci-dessous un fichier de configuration snmpd.conf/net-snmp minimaliste :
```
Comment (cg-tw) [inline fr/getting-started/monitor-linux-server-with-snmp.md:19]:
```suggestion
Voici ci-dessous un fichier de configuration snmpd.conf/net-snmp minimaliste :

```

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: fr/getting-started/monitor-linux-server-with-snmp.md [hunk 21]
```diff
@@ -0,0 +1,108 @@
...
+```shell
+systemctl restart snmpd
+systemctl enable snmpd
+```
+
+> Le serveur cible doit être accessible depuis le collecteur Centreon sur le port SNMP UDP/161.
+
+### Sur le collecteur
+
+1. Connectez-vous à votre collecteur en SSH et installez le plugin SNMP Linux (voir la [procédure de surveillance pour le Plugin Pack **Linux SNMP**](../integrations/plugin-packs/procédures/operatingsystems-linux-snmp.html) pour plus d'informations) :
```
Comment (cg-tw) [inline fr/getting-started/monitor-linux-server-with-snmp.md:55]:
```suggestion
1. Connectez-vous à votre collecteur en SSH et installez le plugin SNMP Linux (voir la [procédure de surveillance pour le Plugin Pack **Linux SNMP**](../integrations/plugin-packs/procedures/operatingsystems-linux-snmp.html) pour plus d'informations) :

```

**Review by cg-tw — CHANGES_REQUESTED** [review]

**Review by cg-tw — APPROVED** [review]

File: fr/getting-started/monitor-linux-server-with-snmp.md [hunk 22]
```diff
@@ -0,0 +1,108 @@
...
+
+   ![image](../assets/getting-started/quick_start_linux_4a.png)
+
+   Vous pouvez également utiliser le raccourci situé à côté du nom de l'hôte pour accéder directement à la page **Configuration > Services > Services par hôte**. La liste sera filtrée par le nom de l'hôte :
+
+   ![image](../assets/getting-started/quick_start_linux_4b.png)
+
+   ![image](../assets/getting-started/quick_start_linux_5.png)
+
+6. [Déployez la configuration](first-supervision#deploying-a-configuration).
```
Comment (cg-tw) [inline fr/getting-started/monitor-linux-server-with-snmp.md:98]:
```suggestion
6. [Déployez la configuration](first-supervision#deployer-une-configuration).

```

**Review by cg-tw — CHANGES_REQUESTED** [review]

**Review by cg-tw — APPROVED** [review]

**Review by pkriko — APPROVED** [review]

**Comment by cg-tw** [comment]
closed by https://github.com/centreon/centreon-documentation/pull/1323

**Pull request closed without merging**